dbt/include/athena/macros/materializations/models/incremental/incremental.sql

```diff
@@ -6,6 +6,7 @@
   {% set on_schema_change = incremental_validate_on_schema_change(config.get('on_schema_change'), default='ignore') %}
 
   {% set lf_tags = config.get('lf_tags', default=none) %}
+  {%- set lf_tags_columns = config.get('lf_tags_columns', default=none) -%}
```

Comment:
picky me, why this has `{%-` and the other `{%` maybe you could uniform it for consistency:)